xfields/ibs/_equilibrium.py

```diff
@@ -230,9 +230,9 @@ def compute_emittance_evolution(
             (e.g. from a feedback system) and is determined from the horizontal
             emittance based on the value of `emittance_coupling_factor`. In this
             case the total transverse emittance is NOT preserved.
-        Providing `None` or an empty string allows one to study a scenarion without
-        constraint. Note as `emittance_coupling_factor` defaults to 0, this parameter
-        has no effect unless a non-zero value is provided.
+        Providing `None` allows one to study a scenario without constraint. Note
+        that as `emittance_coupling_factor` defaults to 0, the constraint has no
+        effect unless a non-zero factor is provided.
```

Comment:
I don't accept "", better for the checks below. One can just as simply provide `emittance_constraint=None` (which personnally I also find cleaner and more explicit)